crates/subspace-core-primitives/Cargo.toml

```diff
@@ -44,19 +44,25 @@ criterion = "0.4.0"
 [features]
 default = [
     "parallel-decoding",
+    "serde",
     "std",
 ]
 # Parallel decoding will use all CPUs available, but will allocate a memory of a size of a sector instead of square root
 # of that
 parallel-decoding = ["rayon"]
+serde = [
+    "dep:serde",
+    # TODO: `serde_arrays` doesn't support `no_std` right now: https://github.com/Kromey/serde_arrays/issues/8
+    "dep:serde_arrays",
```

Comment:
@nazar-pc Since you have marked `serde_arrays` with `dep:` we won't be able to use `serde_arrays` feature in conditional compilation.

Reply:
As mentioned in the other PR that is the whole point here. To make sure all serde-related features get enabled at the same time or not at all. There is no value in `serde_arrays` feature in conditional compilation.